code/js/modules/Sitelist.js

```diff
@@ -80,6 +80,7 @@
       "duckburgradio": { name: "Duckburg Radio", url: "http://beta.radio-mb.com", controller: "DuckburgRadioController.js", alias: ["radio-mb"] },
       "earbits": { name: "Earbits", url: "http://www.earbits.com" },
       "egghead": { name: "egghead.io", url: "https://egghead.io" },
+      "embedded": { name: "Embedded Element", url: "http://www.example.com", controller: "EmbeddedElementController.js", alias: ["example"] },
```

Comment:
Can you make the default url `null` or `""` and drop the alias? (I think) it should work without those properties and it makes this line confusing imo.